.github/pull_request_template.md

```diff
@@ -0,0 +1,43 @@
+# Pull Request Summary
+(Instructions: this, and all subsequent sections of text should be removed and filled in as appropriate.)   
+Please describe the PR summary
+
+## Description
+Provide a detailed description of what this PR does.
+What bug does it fix, or what feature does it add?
+Is a change of answers expected from this PR?
+
+### Issue(s) addressed
+* Is there an issue associated with this development (bug fix, enhancement, new feature)?    
+Please add a reference to a related issue(s) in WW3 repository.
+Link the issues to be closed with this PR, whether in this repository, or in another repository.
```

Comment:
Maybe give a link to this GitHub page:
https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue
Note that properly "linked issues" (either automatic links, or manual ones using the correct keywords) will be automatically closed when the PR is merged.

Reply:
@ukmo-ccbunney Done.